objectRecognitionSource/src/main/java/edu/ml/tensorflow/ObjectDetector.java

```diff
@@ -45,6 +42,11 @@
     // "year-month-date-hour-minute-seconds"
     private static SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd--HH-mm-ss");
     
+    // Getting meta config options for YOLO Processor
+    public MetaBasedConfig metaConfigOptions = new MetaBasedConfig();
+    private final float MEAN = metaConfigOptions.mean;
```

Comment:
Just getting a chance to read this. So instead here, I should just leave the original MEAN as a constant taken from the Config file (how it was before), and not set it as part of the MetaBasedConfig.

Let me know if that seems about right.

Reply:
@namirfa Yes, I think so.  I don't see any MEAN/mean/Mean/etc/ in any of the `.meta` or `.cfg` files, so I don't think we're gathering it from anywhere.  It's just some number that's meaningful.  Somehow?

I also looked for variations on `scale` and they don't exist either.  From some googling, I found some hints [here](https://github.com/thtrieu/darkflow/issues/38).  They appear to say that the 255 is used for normalizing the images -- I suspect that it has to do with the scaling of colors or B&W it's using.  The hint (link above) referred to that scaling number being in the range (0, 1) when running on the android version of things, so I that the 255 has to do with the number of shades involved or or something else referring to greyscale issues or something (but that's really more speculation).  It seems to be structural part of YOLO v2 (at least) that we probably shouldn't change.  It might make sense to change the name to NORMALIZATION_SCALE or something so we don't have to re-derive it each time :-).  Or maybe just add a comment to the Config file that says what it's for.  

If you read thru the graph builder & use thereof in `ObjectDetector.createNormalizerGraph(), it's decoding the JPEG into some matrix (I think), and then dividing that whole matrix (or, really, TensorFlow `Output` object) by the 255/Mean/Scale value.  So, I think it's taking the values and normalizing them into an 8-bit color/B&W scale.  I don't really think we want it to be adjustable -- and if someone wants to, they can change the Config.java file.  

So, bottom line.  Maybe I've convinced myself that it'll work as is -- pass it straight through from Config -- Don't try & read it from `.meta` (which I don't think you're doing).  Whether it has to go via `MetaBasedConfig` isn't clear.  Probably so to keep it clean, but please add a comment that says that this is not read from the `.meta` file.